core/src/main/java/de/flo56958/MineTinker/Utilities/ChatWriter.java

```diff
@@ -19,8 +20,24 @@
 
     public static final String CHAT_PREFIX;
 
+    private final static TreeMap<Integer, String> map = new TreeMap<>();
+
     static {
     	CHAT_PREFIX = Main.getPlugin().getConfig().getString("chat-prefix");
+
+        map.put(1000, "M");
```

Comment:
maybe add the bold Numbers as well. if somebody really wants to have the exp written as roman.
for 25k exp it would be XXV in bold instead of MMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMM
(much more readable)

Reply:
Added underlines to be more consistent with how larger numerals are commonly displayed (line above numeral)